devauth/devauth.go

```diff
@@ -1741,6 +1741,14 @@ func (d *DevAuth) updateCheckInTime(
 				err = errors.Wrap(err, "reindex reporting job error")
 				return
 			}
+		} else {
+			// update check-in time in inventory
+			if err := d.syncCheckInTime(ctx, checkInTime, deviceId, tenantId); err != nil {
+				log.FromContext(ctx).Errorf(
+					"failed to synchronize device check-in time with inventory: device %s: %s",
+					deviceId, err.Error(),
+				)
+			}
```

Comment:
I'm not sure if this should be conditional on the reporting service being enabled, because it will eventually break once we get the reporting service.

Reply:
The idea here is that the check-in time is provided by device search API served by reporting, but if the reporting is not available, the check-in time is provided by inventory. The UI will use reporting to get check-in time if available and inventory if not.
I don't think we need/want to sync check-in time with both - reporting and inventory.
Does it make sense?

Reply:
I get the idea, but the data will still be there in inventory search once we enable reporting (disable this feature).

Reply:
I don't think it make sense to sync check-in time both with reporting and inventory, once/if we'll introduce reporting.
If we'll decide to not reintroduce reporting, then we can remove the condition.
Once/if we'll be reintroducing reporting with its device search API, we can prepare migration or command which will drop check-in time attributes from the devices in inventory.
What do you think?